test/CMakeLists.txt

```diff
@@ -3,8 +3,8 @@ if (check)
   set(SOURCES
     runner.c
     tsclient.c
-    testcases/pgagroal_test_1.c
-    testcases/pgagroal_test_2.c
+    testcases/test_connection.c
```

Comment:
This is not ideal. You should be free to add any test_xxx.c without modifying the CMakeLists. Try what pgmoneta does:
```
FILE(GLOB_RECURSE SOURCE_FILES "libpgmonetatest/*.c" "testcases/*.c")
FILE(GLOB_RECURSE HEADER_FILES "include/*.h")
set(SOURCES ${SOURCE_FILES} ${HEADER_FILES})
add_executable(pgmoneta-test runner.c ${SOURCES})
```